code/modules/lighting/lighting_system.dm

```diff
@@ -49,10 +52,8 @@
 		CRASH("The first argument to the light object's constructor must be the atom that is the light source. Expected atom, received '[A]' instead.")
 	..()
 	owner = A
-	radius = A.luminosity
-	__x = owner.x
-	__y = owner.y
-	SSlighting.changed_lights |= src
+	UpdateLuminosity()
+	changed()
```

Comment:
This changed should probably be removed since UpdateLuminosity already calls it.